doc/configuration.rst

```diff
@@ -446,10 +446,10 @@ worker-disconnect-delay
 [spark]
 -------
 
-Parameters controlling the default execution of ``SparkSubmitTask``:
+Parameters controlling the default execution of ``SparkSubmitTask`` and ``PySparkTask``:
```

Comment:
minor note here is you can use the syntax `:py:class:`~luigi.contrib.spark.PySparkTask`` to get inline links to the class definition